forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function()
+{
+    $(".qa-form-light-button-flag").attr("type", "button");
+    
+    $(".qa-form-light-button-flag").click( function()
+    {
+        var postId = $(this).data("postid");
+        var postType = $(this).data("posttype");
+        var parentId = $(this).data("parentid");
+        
+        $("#flagbox-popup").show();
+        
+        $(".qa-flag-reasons-wrap .closer").click( function()
+        {
+            $("#flagbox-popup").hide();
```

Comment:
To samo co w linijkach 3, 5 i 11 - element z DOM pobieraj raz, przypisuj do zmiennej i dopiero na zmiennej działaj.